jetstream/statistics.py

```diff
@@ -482,7 +482,7 @@ def _make_grid(values: Series, size: int, attempt_geometric: bool) -> MakeGridRe
     if geometric and start == 0:
         start = values.drop_duplicates().nsmallest(2).iloc[1]
         assert start != 0
-    f = np.geomspace if geometric else np.linspace
+    f: Any = np.geomspace if geometric else np.linspace
```

Comment:
There might be a better way to make `mypy` happy here?

Reply:
I tried `f: Callable[[int, int, int], np.ndarray] = np.geomspace if geometric else np.linspace` and got `error: Incompatible types in assignment (expression has type "function", variable has type "Callable[[int, int, int], ndarray]")` for my troubles so who knows, `Any` seems fine